src/app/layouts/layouts-routes-data.ts

```diff
@@ -30,6 +30,7 @@ export const layoutsRoutesData = {
     "card-sample-2": { displayName: "Horizontal Card", parentName: "Card" },
     "card-sample-3": { displayName: "Horizontal Card Alt", parentName: "Card" },
     "card-sample-4": { displayName: "Other Examples", parentName: "Card" },
+    "card-styling-sample": { displayName: "Styled card", parentName: "Card" },
```

Comment:
Consider using a capital C for "card". All other card's samples are named "Card".

Reply:
Done